deployments/epinio.go

```diff
@@ -371,7 +377,7 @@ func (k *Epinio) createIngress(ctx context.Context, c *kubernetes.Cluster, subdo
 										}}}}}}},
 				TLS: []networkingv1.IngressTLS{{
 					Hosts:      []string{subdomain},
-					SecretName: "epinio-tls",
+					SecretName: EpinioDeploymentID + "-tls",
```

Comment:
Are you preparing Epinio for multiple deployments? Maybe/Eventually we should invest in a function "names.TLSSecretSomething" instead of constructing the name every time.

Reply:
I just remember that in some places we construct the name from the deployment name and here we had that hardcoded. I changed it just to make sure, if we ever rename the deployment, that this secret name will also be changed.

Maybe it deserves a bigger refactoring like the one you describe. Not sure if we want to do it here.

Reply:
Nah, let's do it later